paasta_tools/long_running_service_tools.py

```diff
@@ -244,26 +244,35 @@ def get_healthcheck_mode(
     def get_bounce_start_deadline(self) -> float:
         return self.config_dict.get("bounce_start_deadline", 0)
 
+    def get_autoscaled_instances(self) -> int:
+        try:
+            zk_instances = get_instances_from_zookeeper(
+                service=self.service, instance=self.instance
+            )
+            log.debug("Got %d instances out of zookeeper" % zk_instances)
+            return zk_instances
+        except NoNodeError:
+            log.debug("No zookeeper data, returning None")
+            return None
+
+    def set_autoscaled_instances(self, instance_count: int) -> None:
+        """Set the number of instances in the same way that the autoscaler does."""
+        set_instances_for_marathon_service(
+            service=self.service, instance=self.instance, instance_count=instance_count,
+        )
+
```

Comment:
This is unused, for now  -- the idea being that @enochtangg can make the autoscaling API endpoint use this / the KubernetesDeploymentConfig version of it.

Reply:
@EvanKrall Just curious, would it make sense to keep these methods in `MarathonServiceConfig()`, and replicate the same functionality in `KubernetesServiceConfig()`? Then just have the parent class `LongRunningServiceConfig()` as not implemented? Not sure how the convention was set up here, but do we want marathon/k8s specific methods in LongRunningServiceConfig anyways?

Reply:
Yeah, I think that makes sense, though the native scheduler uses this, and it looks less delete-able than I thought. Lemme do a quick refactor.

Reply:
@enochtangg k I moved this stuff in d2e209dd6